first formatting and clarification of MathML (content/presentation)

kohlhase opened the pull request:
In response to Fred's request for something on content MathML here I have clarified the role of MathML in the manifesto. 
I have also added some MD formatting to the document overall. I guess that I will continue polishing this today, but wanted to get off a pull request early to get feedback.

physikerwelt commented on manifesto.md:
```diff
@@ -5,56 +5,75 @@ title: MathML Manifesto
+static formulas, it must be provided in the source code of web pages or
+electronic mails in order to make it usable by the largest number of tools,
+including web crawlers and browsers without support for script language such
+as Javascript.
```
it's JavaScript

kohlhase replied:
I have fixed that in the new commit

physikerwelt commented on manifesto.md:
```diff
@@ -5,56 +5,75 @@ title: MathML Manifesto
-   as Javascript.
+Modern mathematical language relies on mathematical formulae, highly structured
+expressions composed of previously introduced identifiers that are presented in a
+conventionalized sesquidimensional notation.
```
What is a sesquidimensional notation?

kohlhase replied:
sesqui-something -is 1.5-something. So "sesquidimensional notation" is "1.5-dimensional notation"

physikerwelt commented:
:+1: (looks good to me)

kohlhase commented:
I have added some more formatting, structure, and consideration for content MathML, if you agree, merge the pull request please.

fred-wang commented:
Thanks Michael. The meaning of "sesquidimensional" is also not obvious to me but otherwise that looks good to me.

kohlhase commented:
> The meaning of "sesquidimensional" is also not obvious to me

I could add an explanation in brackets. Would that help?

kohlhase commented:
I am also not so happy about the overall form of the "manifesto". It is simply not of the form of a manifesto. I am considering adding the "MathML elevator pitch" as a preamble. 

```
The MathML association is a charitable organization under the Not-for-Profit Corporation Law of the State of New York, with the following goals:

- Native implementation of the MathML language in all Web rendering engines.
- Development of software tools to let users read, write, search, store and transfer MathML formulas; with particular attention to make them accessible to people with disabilities.
- Usage of the MathML language to publish educational, technical and scientific content in Web sites, ebooks or Web apps.
```

What do you think? @dginev @r-gaia-cs please also comment.

fred-wang commented:
> I could add an explanation in brackets. Would that help?

My preference would be that the manifesto is easy to read and understandable by a general audience.

I'm not sure what is meant by "1.5-dimension" (why not 2-dimensional) and whether it's really necessary to mention that.

kohlhase commented:
> I'm not sure what is meant by "1.5-dimension" (why not 2-dimensional)

math formulae are essentially linear/horizontal (1-dimensional) like text, but have a (limited) vertical component. So they are not really 2-dimensional but (by agreement at least in the Math WG) 1.5-dimensional.

kohlhase commented:
> whether it's really necessary to mention that.

not really necessary, but it shows some of the complexity.

fred-wang commented:
> I am also not so happy about the overall form of the "manifesto". It is simply not of the form of a manifesto. I am considering adding the "MathML elevator pitch" as a preamble. 

Yes, I agree a preamble is also necessary. But instead of using the "MathML association" elevator pitch, I would say something like "Blah blah Here are the problems blah blah As a consequence here is what should be done to address them blah blah Hence we, MathML enthusiasts, Web engine developers, blah blah engage to blah blah".

kohlhase commented:
> Yes, I agree a preamble is also necessary. But

You are right, I will try to draft something today unless you want to do that.

fred-wang commented:
> You are right, I will try to draft something today unless you want to do that. 

Please feel free to do so. Thanks.

fred-wang commented:
I think we can reformulate the fact that the visual representation of mathematics is complex, without mentioning "sesqui-" or "1.5-" dimensional.

kohlhase commented:
> I think we can reformulate the fact that the visual representation of mathematics is complex, without mentioning "sesqui-" or "1.5-" dimensional.

I will try that as well.

kohlhase commented:
I have added a preamble in the spirit Fred suggested and simplified the "sesqui" paragraph. I think that the pull request is ready for merging (at least I will not work on it any more; other duties await).

Please read carefully. 

BTW, I think the manifest would deserve a top-level spot in the web site.

physikerwelt commented:
(off-topic) I still don't get why 0,5 dimensions are missing. I used to introduce presentation mathml as as declarative language that defines the arrangement of mathematical symbols on a twodimensional surface. I also thought about the visual impression on non-flat surfaces, but I never had any doubts about the dimensionality. Probably the 1.5 dimensions originate from a non gemetric, information driven approach and have no relation to the actual rendering (for example as SVG). If a standard Ascii text is one dimensional over the domain of Ascii characters, would pmml be a tuple consisting of the mathematical character and the horizontal position information. If that was the case how to distingush between $a^b_c$ and ${a^b}_c$. However, given a 1.5 dimensional background the line-breaking problem becomes much more a rendering problem and not a transformation from one pmml-expression to another one.

rgaiacs commented:
Thanks @kohlhase. It looks good to merge.

kohlhase commented:
> (off-topic) I still don't get why 0,5 dimensions are missing. 
> the only (empirical) finding expressed by this metaphor is that formulae can get arbitrarily long (that is one dimension) but not arbitrarily high (half a dimension). 

This is only a metaphor, not a precise definition. If you do not like this metaphor, that is OK, this is something the Math WG could relate to.

dginev commented:
I guess the Math WG is not a big fan of matrices, which are beautifully 2-dimensional. But math-in-text is definitely bound to being no more than twice the actual line height, otherwise the manuscript becomes extremely ugly. So there are reasonable upper bounds there for sure.

fred-wang commented:
OK, I think we can merge that if nobody objects. And now that sesqui-\* has gone, we won't lose more time on "philosophical" debates, internally or with the outside world.

The last commit adds a HTML comment "LocalWords:". I'm not sure if it was left on purpose, it should probably be removed before merging. BTW, if the question was whether we should use American English, I guess the answer is yes since we (or rather the "MathML association" as a moral entity) is American. We could also provide non-official localized versions of our documents, but that will be more work, so no need to consider that for now...

physikerwelt commented:
Thank you. However, I have the feeling to loose time whenever I'm not in a
philosophical debate;-)
On Jun 9, 2015 8:36 PM, "Frédéric Wang" notifications@github.com wrote:

> Merged #9 https://github.com/MathML/website/pull/9.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/MathML/website/pull/9#event-326622353.
